src/items/menu.js

```diff
@@ -138,8 +143,13 @@ export default class MenuPanelItem extends PanelItem {
           } else if (action === 'logOut') {
             logout(false);
           }
-        }
+        },
+        onclose: () => setTimeout(() => {
```

Comment:
Why is this `setTimeout` here ?

Reply:
It's needed because the global click event handler (whether it is on the menu or anywhere else) immediately triggers `hide()`, so that `onclose` is called before this `onclick`; so without the `setTimeout` the `onclick` will *always* think that the menu is inactive, since `onclose` is called on the same tick and just before the `onclick`.

Classic asynchronous programming "callback hell".

Reply:
If it is really needed, please explicitly add the radix argument.

Reply:
You mean the timeout? As in `setTimeout(..., 0)`?

Reply:
Yes, however, the minimum possible value is `1`.

Reply:
Sorry, yes -- the actual timeout. I had `parseInt` in my mind for some reason.

Reply:
According to MDN, 0 is the minimal value, which means "execute as soon as possible", which is what it should actually do.

Reply:
(This is also pretty idiomatic from my experience with JS.)

Reply:
Huh. Not sure why I was under the impression that the lowest value was one. Not that setTimeout is accurate in any way anyhow :laughing:

Reply:
Ok, done.